datafusion-cli/Cargo.toml

```diff
@@ -31,7 +31,7 @@ rust-version = "1.58"
 clap = { version = "3", features = ["derive", "cargo"] }
 rustyline = "9.0"
 tokio = { version = "1.0", features = ["macros", "rt", "rt-multi-thread", "sync", "parking_lot"] }
-datafusion = { path = "../datafusion", version = "7.0.0" }
+datafusion = { path = "../datafusion", version = "7.1.0" }
 arrow = { version = "9.0.0" }
 ballista = { path = "../ballista/rust/client", version = "0.6.0" }
 env_logger = "0.9"
```

Comment:
To unify the style, I think we can add a new line?

Reply:
Either is fine with me -- I think some editors are configured to remove trailing newlines by default so perhaps that is what happened here?

Reply:
yes, some editor may remove trailing newlines. Either is fine with me too.

this file didn't have newline at the end, so I just let it be
https://github.com/apache/arrow-datafusion/blob/maint-7.x/datafusion-cli/Cargo.toml